### PR TITLE
QA-HEAL - update to 2022.02

### PR DESCRIPTION
**Pull request by haraprasadj**
Link to Jira ticket if there is one:

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## :warning: Services on branch
- metadata

File: qa-heal.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -8,27 +8,26 @@
...
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2022.01",
-    "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.01",
-    "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2022.01",
-    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2022.01",
-    "frontend-framework": "quay.io/cdis/frontend-framework:main",
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.01"
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2022.02",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2022.02",
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2022.02",
+    "metadata": "quay.io/cdis/metadata-service:fix_pdaps",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:20]: this branch should be included in 1.6.3

File: qa-heal.planx-pla.net/manifest.json [hunk 2]
```diff
@@ -8,27 +8,26 @@
...
-    "peregrine": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/peregrine:2022.01",
-    "pidgin": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/pidgin:2022.01",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:2022.01",
-    "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.01",
-    "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.01",
-    "ssjdispatcher": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/ssjdispatcher:2022.01",
-    "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.01",
-    "spark": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/gen3-spark:2022.01",
-    "tube": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/tube:2022.01",
-    "frontend-framework": "quay.io/cdis/frontend-framework:main",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:30]: put this back?

Comment (haraprasadj) [reply qa-heal.planx-pla.net/manifest.json:30]: yeah, let me do that, it should not impact testing of 2022.02 versions anyway

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## :warning: Services on branch
- frontend-framework

File: qa-heal.planx-pla.net/manifest.json [hunk 3]
```diff
@@ -8,27 +8,27 @@
...
-    "wts": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/workspace-token-service:2022.01"
+    "hatchery": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/hatchery:2022.02",
+    "indexd": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexd:2022.02",
+    "manifestservice": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/manifestservice:2022.02",
+    "metadata": "quay.io/cdis/metadata-service:1.6.3",
```
Comment (mfshao) [inline qa-heal.planx-pla.net/manifest.json:20]: ECR has this one right?

Comment (haraprasadj) [reply qa-heal.planx-pla.net/manifest.json:20]: let me check

**Review by mfshao — APPROVED** [review]
👍

**Comment by PlanXCyborg** [comment]
# qa-heal.planx-pla.net/manifest.json
## :warning: Services on branch
- frontend-framework

**Pull request merged**